rb/lib/selenium/webdriver/common/error.rb

```diff
@@ -45,7 +52,7 @@ class WebDriverError < StandardError; end
 
       class NoSuchElementError < WebDriverError
         def initialize(msg = '')
-          super("#{msg}; #{SUPPORT_MSG} #{ERROR_URL}#no-such-element-exception")
+          super("#{msg}; #{SUPPORT_MSG} #{URLS[:NoSuchElementError]}")
```

Comment:
Would it make sense to move this to WebDriverError constructor?

```ruby
if (url = URLS[self.class.name])
  msg = "#{msg}; #{SUPPORT_MSG} #{url}"
end
super(msg)
```
super()

Reply:
I was thinking the same, the reason why I didn't do it is that once we have all the pages added in the documentation, we will just refactor the WebDriverError constructor to be:

```Ruby
def initialize(msg = '')
  super("#{msg}; #{SUPPORT_MSG} #{URLS[self.class.name]}")
end
```
And that probably will be done by the last person updating the docs 

So Instead of having to add a conditional and remove it, I was thinking that is clearer for future maintainers to notice that there is a code duplication, but if you think is clearer with the conditional, I can just add it :) and a comment for the eventual removal

Reply:
@p0deje I updated it now with a different style to avoid rubocop complaints and with a comment so in the future someone will remember to remove it